src/gnc_attitude_estimator.cpp

```diff
@@ -282,15 +282,15 @@ static void ukf(AttitudeEstimatorState &state, AttitudeEstimatorData const &data
     // Plus the associated covariances
     UkfVector6 dx1 = state.sigmas[0] - x_bar;
     UkfVector5 dz1 = state.measures[0] - z_bar;
-    P_bar = P_bar + weight_c * dx1 * lin::transpose(dx1);
-    P_vv = P_vv + weight_c * dz1 * lin::transpose(dz1);
-    P_xy = P_xy + weight_c * dx1 * lin::transpose(dz1);
```

Comment:
Here's the boneheaded mistake where I wasn't zeroing out `P_bar`, `P_vv`, and `P_xy` on each iteration.